python/gsw/main.py

```diff
@@ -81,8 +89,6 @@ def check_for_email(self):
             #.fetch() fetches the mail for given id where 'RFC822' is an Internet 
             # Message Access Protocol.
             _, data = self.mail.fetch(num,'(RFC822)')
-            #mark the email message as read.
-            self.mail.store(num, '+FLAGS', '(\\Seen)')
```

Comment:
Why did you stop marking emails as read?

Reply:
It didn't make a difference. Once the function reads the email message, the "read" flag in the email message is automatically set to True anyways.

Reply:
Gotcha.